api/http/api_deployments.go

```diff
@@ -352,7 +352,6 @@ func (d *DeploymentsApiHandlers) newImageWithContext(ctx context.Context, w rest
 
 	// parse multipart message
 	multipartUploadMsg, err := d.ParseMultipart(formReader)
-	defer r.MultipartForm.RemoveAll()
```

Comment:
I am not sure about that, why are we dropping that?

Reply:
This statement is a NOOP. We're not calling `r.ParseMultipartForm` anymore.